qkernel/src/lib.rs

```diff
@@ -474,6 +482,32 @@ fn InitLoader() {
     LOADER.InitKernel(process).unwrap();
 }
 
+#[cfg(feature = "cc")]
+//Need to initialize PAGEMGR(pagepool for page allcator) and kernel page table in advance
+fn InitShareMemory() {
+    *GHCB[0].lock() = Some(GhcbHandle::default());
```

Comment:
I see this is a repeated pattern. Maybe have some accessor methods for **GHCB** elements, that way you can also do sanity check on indexes.

Reply:
>  sanity check on indexes.

an assert would do, but that rust runtime will panic on out-of-bound anyways, that's the same effect. I think this depends on whether you consider this operation failiable.